### ISSUE TITLE
False positive S4653 when unit is computed with Sass arithmetic

### ISSUE THREAD

**Issue by nicolas-harraudeau-sonarsource** [SonarSource/sonar-css#160]
Reproducer: In a Sass file (*.scss)
```
$i: 1;

.parent {
    width: $i/1.3+px; // valid but raises an issue
    height: $i/1.3+0px; // valid but raises an issue

    height: $i/1.3*1px; // this is OK and no issue is raised, recommended workaround
}

```

Note: The `+px` contactenation can create other kind of bugs, it is out of scope of this ticket. See [this link](https://css-tricks.com/snippets/sass/correctly-adding-unit-number/) for more information.

Source: Bug report from pgregory [on Community](https://community.sonarsource.com/t/false-positive-for-scss/8388/4)

**Comment by vilchik-elena** [comment]
https://github.com/stylelint/stylelint/issues/4098

**Comment by guillaume-dequenne-sonarsource** [comment]
FP documented in ruling test. Still not fixed (requires fix in the parser used by stylelint).
